gyp/pylib/gyp/generator/make.py

```diff
@@ -2469,16 +2484,27 @@ def CalculateMakefilePath(build_file, base_name):
         header_params.update({"link_commands": LINK_COMMANDS_ANDROID})
     elif flavor == "zos":
         copy_archive_arguments = "-fPR"
-        makedep_arguments = "-qmakedep=gcc"
+        CC = GetEnvironFallback(("CC_target", "CC"), "njsc")
+        if CC == "clang" or CC == "ibm-clang64":
+            makedep_arguments = "-MMD"
+            CC_host = GetEnvironFallback(("CC_host", "CC"), "clang")
+            CXX_target = GetEnvironFallback(("CXX_target", "CXX"), "clang++")
+            CXX_host = GetEnvironFallback(("CXX_host", "CXX"), "clang++")
```

Comment:
If CC is set to `ibm-clang64` and CXX is left unset, then it seems like CXX will still fall back to `clang++`. Could that be a potential issue? Should we make sure it fallbacks to `ibm-clang++64` instead?